src/createStore/createStore.js

```diff
@@ -11,11 +11,21 @@ const defaultOptions = {
   listeners: undefined,
   devtools: true,
   name: 'store',
+  drivers: {
+    http: (...args) => fetch(...args),
```

Comment:
Why do you define a new function instead of passing directly `fetch` ?

Reply:
so you can overload it and have a standardized name.

Reply:
I can do `http: fetch` yes.
 - But in the lib I want to have something that dispatch classics events from http (STARTED/LOADING/ENDED/ERROR)